aries-site/src/pages/index.js

```diff
@@ -1,69 +1,100 @@
-  if (featured.includes(card.name)) featuredCards.push(card);
-  else cardOrder.push(card);
-});
+const Intro = ({ children }) => {
+  const size = useContext(ResponsiveContext);
+  return (
+    <Box height={{ min: 'medium' }} justify="center">
+      <Grid gap="large" columns={size === 'small' ? ['auto'] : ['2/3', 'auto']}>
+        <Card background="none" elevation="none">
+          {children}
+        </Card>
+        {size !== 'small' && (
+          <Card background="none" elevation="none" height="small" />
+        )}
+      </Grid>
+    </Box>
+  );
+};
 
-featuredCards.sort(
-  (a, b) => featured.indexOf(a.name) - featured.indexOf(b.name),
-);
+Intro.propTypes = {
+  children: PropTypes.node,
+};
 
-cardOrder.splice(0, 0, ...featuredCards);
+const Index = () => {
+  const size = useContext(ResponsiveContext);
+  const backgroundImage = {
+    src: { dark: '/home.svg', light: '/home.svg' },
+    alt: 'HPE Design System',
+    margin: { top: '75px', left: '0px' },
+    style: {},
+    position: size !== 'small' ? 'right' : 'left',
+  };
 
-const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/cards-dark.svg', light: '/cards.svg' },
-      alt: 'HPE Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useGrid: true,
-    }}
-    title={title}
-    isLanding
-  >
-    <Meta title={title} description={pageDetails.seoDescription} />
-    <Box gap="large">
-      <PageIntro>
-        <Box justify="center" fill>
-          <Heading margin="none">HPE Design System</Heading>
-          <Paragraph size="large" fill>
-            The Design System was created to empower designers, developers, and
-            others to contribute to making great experiences for the customer.
-          </Paragraph>
+  return (
+    <Layout backgroundImage={backgroundImage} title={title} isLanding pad={{}}>
+      <Meta title={title} description={pageDetails.seoDescription} />
+      <Box>
+        <Intro position="left">
+          <Box pad={calcPad(size)}>
+            {size === 'small' && (
+              <Card background="none" elevation="none" height="small" />
+            )}
+            <Heading margin="none" size="large">
+              Design, develop and deliver
+            </Heading>
+            <Paragraph size="xlarge">
+              Created to empower designers, developers, and others to quickly
+              create accessible user experiences
+            </Paragraph>
+          </Box>
+        </Intro>
+        <Stack guidingChild="last">
+          <Box background="background-front" margin={{ top: 'xlarge' }} fill />
+          <Featured />
+        </Stack>
+        <WhatIs />
+        <Box fill>
+          <Video controls="below" poster="/static/images/cards.png">
+            <source src="/HPE Design System-v3.mp4" type="video/mp4" />
```

Comment:
Thinking it would be nice to add a `<track />` tag so that we can support subtitles for accessibility.

@MikeWalrath Do you have the script for the video which we could use to make the subtitles?

Reply:
Attaching here! (ignore all design assets)
[HPE Design System V6.pdf](https://github.com/grommet/hpe-design-system/files/5935385/HPE.Design.System.V6.pdf)